src/pages/transparency.tsx

```diff
@@ -47,140 +53,173 @@ export default function WrappingPage() {
-                              text={l('page.transparency.members.wearables_curator_button')} />
-                <ExternalLinkWithIcon href={ABOUT_DELEGATES}
-                              imageSrc={personIcon}
-                              text={l('page.transparency.members.delegate_button')} />
-              </div>
-            </Grid.Column>
-
-            <Grid.Column tablet="12">
-              <div className="TransparencySection">
-                <Card className="TransparencyCard">
-                  {data && data.teams.map((team, index) => {
-                    return <MembersSection
-                      key={[team.name.trim(), index].join('::')}
-                      title={team.name}
-                      description={team.description}
-                      members={team.members}
+                    </Card.Content>
+                    <GrantList
+                      status={ProposalStatus.Enacted}
+                      title={l('page.transparency.funding.proposals_funded_label') || ''}
+                    />
+                    <GrantList
+                      status={ProposalStatus.Active}
+                      title={l('page.transparency.funding.active_grants_label') || ''}
                     />
-                  })}
-                </Card>
-              </div>
-            </Grid.Column>
-          </Grid.Row>
-        </Grid>
-      </Container>
-    </>}
+                  </Card>
+                </div>
+              </Grid.Column>
+            </Grid.Row>
+          </Grid>
+        </Container>
+
+        <Container className="TransparencyContainer">
+          <Grid className="TransparencyGrid" stackable>
+            <Grid.Row columns={2}>
+              <Grid.Column tablet="4">
+                <div>
+                  <Header>{l('page.transparency.members.title')}</Header>
+                  <p>{l('page.transparency.members.description')}</p>
+
+                  <ExternalLinkWithIcon href={ABOUT_DAO_URL}
+                                imageSrc={documentOutline}
+                                text={l('page.transparency.members.about_dao_button')} />
+                  <ExternalLinkWithIcon href={WEARABLE_CURATORS_URL}
+                                imageSrc={personIcon}
+                                text={l('page.transparency.members.wearables_curator_button')} />
+                  <ExternalLinkWithIcon href={ABOUT_DELEGATES}
+                                imageSrc={personIcon}
+                                text={l('page.transparency.members.delegate_button')} />
+                </div>
+              </Grid.Column>
+
+              <Grid.Column tablet="12">
+                <div className="TransparencySection">
+                  <Card className="TransparencyCard">
+                    {data && data.teams.map((team, index) => {
+                      return <MembersSection
+                        key={[team.name.trim(), index].join('::')}
```

Comment:
Do we have a member id here? I don't think it is a good practice to use array index in a component key.

Reply:
And id would be better but we are not updating this list constantly, it won't generate much problems